packages/trader/src/AppV2/Components/AccumulatorStats/__tests__/accumulator-stats.spec.tsx

```diff
@@ -91,7 +91,7 @@ describe('AccumulatorStats', () => {
     });
     test('should set animationClass and isMovingTransition based on rows[0][0] changes', async () => {
         renderAccumulatorState(default_mock_store);
-        jest.advanceTimersByTime(3000);
+        await waitFor(() => jest.advanceTimersByTime(300));
         await waitFor(() => {
```

Comment:
now is waitFor required in this line as we are using it above

Reply:
Sorry, I didn't get what you mean, waitFor was imported here before my changes

Reply:
I meant you can remove the waitFor in line 95, since you added it in line 94.

Reply:
Ohh, sorry, will apply